etna/transforms/math/sklearn.py

```diff
@@ -230,3 +231,7 @@ def _inverse_reshape(self, df: pd.DataFrame, transformed: np.ndarray) -> np.ndar
             [transformed[i * time_period_len : (i + 1) * time_period_len, :] for i in range(n_segments)], axis=1
         )
         return transformed
+
+    def get_regressors_info(self) -> List[str]:
+        """Return the list with regressors created by the transform."""
+        return []
```

Comment:
The list of regressors here depends on the `in_column`, if column from `in_column` is regressor and transform is not inplace, then the out_column is also regressor. See the example of such logic [here](https://github.com/tinkoff-ai/etna/blob/fd7f1582f2b0fb2880411cc5f17460d126a03fbd/etna/transforms/encoders/categorical.py#L156)